### PR TITLE
[WIP] SSD bananas data

### PR DESCRIPTION
**Pull request by xiaotinghe**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_computer-vision/ssd.md [hunk 1]
```diff
@@ -248,7 +249,7 @@ We read the Pikachu dataset we created in the previous section.
 
 ```{.python .input  n=14}
 batch_size = 32
-train_iter, _ = d2l.load_data_pikachu(batch_size)
+train_iter, _, data_dir = d2l.load_data_bananas(batch_size)
```
Comment (astonzhang) [inline chapter_computer-vision/ssd.md:252]: do we really need `data_dir`?

**Comment by mli** [comment]
Job d2l-en/PR-1126/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1126/

**Comment by mli** [comment]
Job d2l-en/PR-1126/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1126/

**Comment by astonzhang** [comment]
@goldmermaid  FYI

**Comment by astonzhang** [comment]
Thanks for the smart and hard work!

**Pull request merged**